app/components/HeaderNotifications/HeaderNotifications.css

```diff
@@ -41,9 +44,10 @@ html[data-theme='dark'] .notifications {
 }
 
 .innerNotification {
+  border-bottom: 1px solid var(--color-mono-gray-3);
+  padding-bottom: 15px;
```

Comment:
Veeery picky, but nice to get in early: please move color and "styling" css to the bottom, and let position/display/size lines come first.

Reply:
fixed